RNTester/e2e/sanity.test.js

```diff
@@ -30,7 +30,7 @@ describe('Sanity', () => {
     await element(by.text('OK')).tap();
   });
 
-  it(`Two buttons with JustifyContent:'space-between' should be tappable`, async () => {
+  it('Two buttons with JustifyContent:\'space-between\' should be tappable', async () => {
```

Comment:
The linter would be happy with double quotes here and then you don't need to escape the inner ones.
```suggestion
  it("Two buttons with JustifyContent:'space-between' should be tappable", async () => {
```